### PR TITLE
close #157: tell users how to receive release announcements

### LINKED ISSUE #157: Clarify how the community should get update notifications
Sent to VarNomen@HUGO-int.org before the new website was released:

> Subject:        Notification list for updates
> Date:   Wed, 8 Nov 2023 14:44:36 +0000
> To:     VarNomen@HUGO-int.org <VarNomen@HUGO-int.org>
>
> Apologies if this is the wrong email, but I could not find an
> appropriate alternative on your website.
> Is it possible to be notified of new versions of HGVS being released?
> It would be very helpful in our clinical reporting.

Action: Clarify on the new website where we will announce updates

### PR DESCRIPTION
**Pull request by reece**
Two changes:

1. on top/home page, add sentence (and slightly reformat) section with email group:

image

2. add sentence to box on versions page:

image

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -15,17 +15,18 @@ The HGVS Nomenclature is administered by the [HGVS Variant Nomenclature Committe
...
+    * Messages sent to
+      [hgvs-nomenclature@googlegroups.com](mailto:hgvs-nomenclature@googlegroups.com)
+      will be distributed to members by email.
+    * You do not need to be a member to send a message to the group. **However,
+      you will not see a reply unless you join the group or manually check the
+      [HGVS Nomenclature](https://groups.google.com/g/hgvs-nomenclature)
+      conversation archive**.
 * See [Community Consultation](consultation/index.md) for proposals, including
-  how to submit a proposal.
+  how to submit a proposal for future versions of HGVS Nomenclature.
```
Comment (jfjlaros) [inline docs/index.md:29]: "... of the HGVS Nomenclature"

File: docs/versions/index.md [hunk 2]
```diff
@@ -4,6 +4,9 @@
 
     Read the [release notes for the current version](21.0.md).
 
+    If you want to receive announcements of updates to HGVS Nomenclature, join the
```
Comment (jfjlaros) [inline docs/versions/index.md:7]: "... to the HGVS Nomenclature"

**Review by jfjlaros — COMMENTED** [review]
This looks fine to me, I only have a couple of small suggestions.

**Review by jfjlaros — APPROVED** [review]
Thank you, I have no further comments.

**Review by ifokkema — COMMENTED** [review]
Besides a missing period at the end of the sentence in `docs/versions/index.md`, it looks good to me!

**Pull request merged**